--- a/meson.build	2024-06-19 21:03:14.825723421 +0000
+++ b/meson.build	2024-06-19 20:51:16.168387607 +0000
@@ -252,10 +252,10 @@
   endif
 endif
 
-if host_os.startswith('linux') and cc.has_function('eeze_disk_function', dependencies: dependency('eeze')) == true
-  config_h.set('HAVE_EEZE_MOUNT', '1')
-  eeze_mount = true
-endif
+#if host_os.startswith('linux') and cc.has_function('eeze_disk_function', dependencies: dependency('eeze')) == true
+#  config_h.set('HAVE_EEZE_MOUNT', '1')
+#  eeze_mount = true
+#endif
 
 if get_option('files') == true
   config_h.set('ENABLE_FILES', '1')
